### DISCUSSION TITLE
Can I run GraphRAG locally?

### DISCUSSION THREAD

**Discussion by MINJIK01** [microsoft/graphrag#708] [Q&A] [answered]
Hello. I am really impressed by GraphRAG and thank you for your contribution.

One thing I would like to ask is about the availability of GraphRAG in a local environment. Do I need to use Azure to utilize GraphRAG, or is it possible to use GraphRAG locally?

Thank you :)

**Comment by natoverse** [comment, marked as answer]
I think you are asking about running local models. Please see #657 for some discussion and a number of linked issues with users discussing their results with various open source models.
